cypress/e2e/court-cases/caseDetails.cy.ts

```diff
@@ -413,6 +414,158 @@ describe("Case details", () => {
       .last()
       .should("have.text", "Bichard02: Portal Action: Resubmitted Message.")
   })
+
+  it("should show triggers tab by default when navigating to court case details page", () => {
+    cy.task("insertCourtCasesWithFields", [{ orgForPoliceFilter: "01" }])
+    const triggers: TestTrigger[] = [
+      {
+        triggerId: 0,
+        triggerCode: "TRPR0010",
+        status: "Unresolved",
+        createdAt: new Date("2022-07-09T10:22:34.000Z")
+      },
+      {
+        triggerId: 1,
+        triggerCode: "TRPR0015",
+        status: "Unresolved",
+        createdAt: new Date("2022-07-09T10:22:34.000Z")
+      }
+    ]
+    cy.task("insertTriggers", { caseId: 0, triggers })
+
+    cy.login("bichard01@example.com", "password")
+
+    cy.visit("/bichard/court-cases/0")
+
+    cy.get(".moj-tab-panel-triggers").should("be.visible")
+    cy.get(".moj-tab-panel-exceptions").should("not.be.visible")
+
+    cy.get(".moj-tab-panel-triggers .moj-trigger-row").eq(0).contains("PR10 / Offence 1")
+    cy.get(".moj-tab-panel-triggers .moj-trigger-row").eq(0).contains("Bail conditions imposed/varied/ cancelled")
```

Comment:
nit: is the extra space in `imposed/varied/ cancelled` intentional? I think it would be good to be consistent with spacing or no spacing